client/src/test/java/com/blackship/battlesheep/fx/controllers/utils/PositionUtilsTest.java

```diff
@@ -0,0 +1,46 @@
+package com.blackship.battlesheep.fx.controllers.utils;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.*;
+
+/**
+ * @author Mateusz Słaboński on 27.07.17
+ * @project
+ */
+public class PositionUtilsTest {
+
+    public static final int BOARD_SIZE = 10;
+
+    @DataProvider
+    public static Object[][] oneDimensionalPositionData() {
+        return new Object[][] {
+                {1, 0, 1},
+                {23, 2, 3},
+                {56, 5, 6},
+                {100, 10, 0}
+        };
+    }
+
+    @Test(dataProvider = "oneDimensionalPositionData")
+    public void testCalculateFromTwoDimensionalPosition(int expectedPosition, int givenX, int givenY) throws Exception {
```

Comment:
May be misleading to use oneDimensionalPositionData in test that calculatesFromTwoDimencionalPosition